src/Pages/Dashboard.js

```diff
@@ -1,53 +1,34 @@
 import React, { useEffect } from 'react';
```

Comment:
I fixed our Dashboard and the routing in Dashboard so the whole Dashboard side of things should be working